sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/exemplar/ExemplarReservoir.java

```diff
@@ -20,9 +20,6 @@
  * An interface for an exemplar reservoir of samples.
  *
  * <p>This represents a reservoir for a specific "point" of metric data.
- *
- * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
- * at any time.
  */
 public interface ExemplarReservoir<T extends ExemplarData> {
```

Comment:
Does this guy need to be part of our public API? Do we want to allow people to create their own custom reservoir implementations? Just making sure we're not opening things up a bit too much before we know this is something that we want people to be able to customize.